### PR TITLE
added task BP-02-003 and BP-02-004

### PR DESCRIPTION
**Pull request by Romainua**

### CODE REVIEW

File: testnet/tasks/BP-01-001/iIc4E10d9I2CG5.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "uid": "iIc4E10d9I2CG5",
+    "api_key_name": "first"
+}
```
Comment (0xcoco) [inline testnet/tasks/BP-01-001/iIc4E10d9I2CG5.json:4]: reviewed

Comment (0xcoco) [inline testnet/tasks/BP-01-001/iIc4E10d9I2CG5.json:4]: reviewed 002

Comment (0xcoco) [inline testnet/tasks/BP-01-001/iIc4E10d9I2CG5.json:4]: reviewed 003

**Comment by Mummyhap** [comment]
reviewed 02002

**Comment by Romainua** [comment]
added task 03-001

**Comment by Mummyhap** [comment]
reviewed 03-001

**Comment by Mummyhap** [comment]
reviewed 03-002

**Comment by Mummyhap** [comment]
reviewed 02-004

File: testnet/tasks/BP-02-003/lEcoKEcqOErV.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "provider": "0x5604315Fb2296E1D279a3aDc9DA432CCe90A8553"
+}
```
Comment (Mummyhap) [inline testnet/tasks/BP-02-003/lEcoKEcqOErV.json:3]: reviewed 02-003

**Pull request closed without merging**